extensions/sql-migration/src/service/contracts.ts

```diff
@@ -484,6 +497,26 @@ export namespace ValidateLoginMigrationRequest {
 	export const type = new RequestType<StartLoginMigrationsParams, StartLoginMigrationResult, void, void>('migration/validateloginmigration');
 }
 
+export namespace ValidateSysAdminPermissionRequest {
```

Comment:
Have we added call for these validation requests in some other PR. Like where we are validating these things, do we need to call them from somewhere in Login Migrations Wizard?

Reply:
Yes, these API's will get called as part of Login migration Run Validation wizard that needs to be added once this PR is merged.